apps/deckboard/deckboard.yml

```diff
@@ -0,0 +1,14 @@
+name: Deckboard
+description: 'Custom shortcut for your PC on your phone.'
+website: 'https://rivafarabi.github.io/deckboard'
```

Comment:
I don't see any link to the code. As per the [contribution guidelines](https://github.com/electron/electron-apps/blob/master/contributing.md), if you're submitting a closed source app, please include evidence of it being built with Electron. Thanks!

Reply:
yeah the project is closed source and I'm a bit confused right not because the community guidelines doesn't explicitly tell you what kind of evidence to be included.

Reply:
So should I add the screenshot (with devtools) here or add it into my repo?

Reply:
![devtools](https://user-images.githubusercontent.com/4427494/36906177-d5f3193a-1e67-11e8-8b6e-97712203adcc.PNG)

Ok here is my screenshot of deckboard app with devtools opened